realm/realm-library/src/androidTestObjectServer/kotlin/io/realm/internal/network/LoggingInterceptorTest.kt

```diff
@@ -204,22 +205,55 @@ class LoggingInterceptorTest {
     }
 
     @Test
-    fun googleTokenLogin_obfuscation() {
+    fun googleTokenLogin_authCode_obfuscation() {
         app = TestApp { builder ->
             builder.httpLogObfuscator(HttpLogObfuscator(LOGIN_FEATURE, AppConfiguration.loginObfuscators))
         }
         testLogger = getLogger()
         val token = "google-token"
 
         try {
-            app.login(Credentials.google(token))
+            app.login(Credentials.google(token, GoogleAuthType.AUTH_CODE))
         } catch (error: AppException) {
             Assert.assertEquals(ErrorCode.INVALID_SESSION, error.errorCode)
         } finally {
             assertMessageExists(""""authCode":"***"""")
         }
     }
 
+
+    @Test
+    fun googleTokenLogin_idToken_noObfuscation() {
+        app = TestApp()
+        testLogger = getLogger()
+        val token = "google-token"
+
+        try {
```

Comment:
There is an `assertFailsWithErrorCode` for this pattern. There is still no functionality to match the error though, so you will still have to inspect the error message with `.also { e -> e.message.contains}` or similar.